src/pages/OverviewDetail/OverviewDetail.jsx

```diff
@@ -0,0 +1,34 @@
+import {Link, useParams} from "react-router-dom";
+import "./OverviewDetail.css"
+import posts from "../../constants/data.json"
+
+
+function OverviewDetail() {
+    const { id } = useParams();
+    const blogPost = posts.find(post => post.id === parseInt(id));
+    console.log(id);
+    console.log(useParams());
+
+
+    if (!blogPost) {
+        return <div className="blogpostNotFound" >Blogpost niet gevonden
+               <Link to="/blogposts">Terug naar de overzichtspagina</Link>
+        </div>;
+
+
+    }
+
+    return (
+        <div className="blogpost">
+            <h1>{blogPost.title} ({blogPost.readTime} minuten)</h1>
```

Comment:
Ik had het ook zo gedaan, maar bij mij werkte de link niet. Ik weet niet of dat bij jou ook zo is, maar in de uitwerkingen staat het net op een andere manier die (in ieder geval bij mij) wel werkt!